client/modules/client_state_machine.js

```diff
@@ -124,8 +124,7 @@ define(function(require) {
 
   class ClientStateMachine extends stateMachine.Machine {
     constructor() {
-      super('ClientStateMachine',
-          new DisplayState(ClientModule.newEmptyModule(0)));
+      super(debug, new DisplayState(ClientModule.newEmptyModule(0)));
```

Comment:
I see what's happening now...

Is this just to avoid the extra code in the base class or is there some other reason? This seems like an unnecessarily complicated interface.